docs/releases/node-releases.md

```diff
@@ -74,13 +74,142 @@ The following versions are no longer peered with by nodes running the active ver
+    * **To best understand how these and other epoch related changes will impact your integration, it is highly recommended that the [Upcoming upgrades > Increased work difficulty section](/releases/network-upgrades#increased-work-difficulty) of the Network Upgrades is carefully reviewed**
+* `active_difficulty` [RPC](/commands/rpc-protocol/#work_validate) and [WebSocket](/integration-guides/websockets/#active-difficulty) will automatically begin returning the higher difficulty threshold for send/change blocks in the `network_minimum` field once the epoch upgrade begins, otherwise the response formats will remain the same
+* **BREAKING CHANGE** [`bootstrap_status`](/commands/rpc-protocol/#bootstrap_status) responses now have `connections` field as an array of connection-related fields and adds an `attempts` field with an area of individual bootstrap attempt details, each including information such as a unique id, type of bootstrap (legacy, lazy) and various other granular information.
+* [`account_info`](/commands/rpc-protocol/#account_info) responses now contain `confirmation_height_frontier` which is the hash of the last confirmed block.
+**----------TODO:** Add brief details about how this is helpful to integrations...
+
+--8<-- "process-sub-type-recommended.md"
+
+#### CLI Updates
+* **NEW** [`--generate_config [node|rpc]`](/commands/command-line-interface/#-generate_config-noderpc) prints sample configuration files to *stdout*
+    * `use_defaults` additional argument to generate uncommented entries (not recommended)
+* **NEW** [`--config`](/commands/command-line-interface/#-config-keyvalue) passes configuration arguments, alternative to setting in the config file
+* **NEW** [`--inactive_votes_cache_size`](/commands/command-line-interface/#-inactive_votes_cache_size) allows adjusting of the cache that holds votes where the block is does not have an action election, default is 16384 votes
+* **NEW** [`--rebuild_database`](/commands/command-line-interface/#-rebuild_database) provides a better compaction method for LMDB. **NOTE:** This requires approximately `data.ldb` file size * 2 in free space on disk.
+* **NEW** [`--compare_rep_weights`](/commands/command-line-interface/#-compare_rep_weights) gives the ability to compare the current ledger voting weight distribution against the hard coded weights provided in the node on release. Useful when attempting to use a downloaded ledger. More details on use on [Ledger Management page](/running-a-node/ledger-management)
+
+---
+
+#### WebSockets
+
+* Updates to WebSocket subscriptions are now allowed on the [`confirmation`](/integration-guides/websockets/#confirmations) topic. With `options` of `accounts_add` and `accounts_del` an existing subscription can now be more easily managed to avoid cancelling and resubscribing or managing multiple subscriptions.
```

Comment:
```suggestion
* Updates to WebSocket subscriptions are now allowed on the [`confirmation`](/integration-guides/websockets/#confirmations) topic. With `options` of `accounts_add` and `accounts_del` an existing subscription can now be more easily managed to avoid resubscribing with a large list of accounts or managing multiple subscriptions.
```

Cancelling to re-subscribe wasn't required.